app/components/three-bounce-spinner/styles.scss

```diff
@@ -0,0 +1,42 @@
+/*Huge thanks to @tobiasahlin at http://tobiasahlin.com/spinkit/ */
+.three-bounce-spinner  {
+  margin: 100px auto 0;
+  width: 70px;
+  text-align: center;
+
+  > div {
+    width: 18px;
+    height: 18px;
+    background-color: #EF4836;
+
+    border-radius: 100%;
+    display: inline-block;
+    -webkit-animation: sk-bouncedelay 1.4s infinite ease-in-out both;
+    animation: sk-bouncedelay 1.4s infinite ease-in-out both;
+  }
+
+  .bounce1 {
+    -webkit-animation-delay: -0.32s;
+    animation-delay: -0.32s;
+  }
+
+  .bounce2 {
+    -webkit-animation-delay: -0.16s;
```

Comment:
it the webkit specific css required? I think its generated via plugin. Can you remove it and test once?

Reply:
It may work without -webkit for recent browsers but, I just thought have it for older version support.